### PR TITLE
Complete feature view field detail to guest

### PR DESCRIPTION
**Pull request by nguyentrieutien1**
- [#TicketID](https://edu-redmine.sun-asterisk.vn/issues/73411)

## WHAT (optional)
- Show fetail detail to guest

## Evidence (Screenshot or Video)
image
image

### CODE REVIEW

**Comment by nguyentrieutien1** [comment]
ready

File: app/controllers/fields_controller.rb [hunk 1]
```diff
@@ -1,9 +1,7 @@
 # frozen_string_literal: true
 
 class FieldsController < ApplicationController
-  def index
-    @field = Field.new
+  def show
+    @field = Field.find params[:id]
```
Comment (manhhnee) [inline app/controllers/fields_controller.rb:5]: Dùng find_by đi anh

**Review by manhhnee — COMMENTED** [review]
comment

**Comment by vanvtt-0952** [comment]
commented
có comment của Mạnh kìa

File: app/controllers/fields_controller.rb (hunk 1 above)
Comment (nguyentrieutien1) [reply app/controllers/fields_controller.rb:5]: 1. Why using find_by instert of find  while you mark index to id column ??
image

2. I'm handle exception  with handle_error_and_redirect hepler, if using find_by, you have to handle nil case.

3. find_by too many the code than find

**Comment by nguyentrieutien1** [comment]
ready

File: app/views/fields/show.html.erb [hunk 2]
```diff
@@ -5,19 +5,8 @@
...
-        <div class="text">
-          <span class="meta">May 20, 2020</span>
-          <h3 class="mb-4"><a href="#">Romolu to stay at Real Nadrid?</a></h3>
-          <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Possimus deserunt saepe tempora dolorem.</p>
-        </div>
-      </div>
-    </div>
+    <% @field.field_types.each do |field_type| %>
+      <%= render field_type %>
+    <% end %>
```
Comment (vanvtt-0952) [inline app/views/fields/show.html.erb:10]:
```suggestion
     <%= render(partial: "field_type", collection: @field.field_types) || "There's no field type to be displayed" %>
```
em nghĩ sao, hoặc bỏ qua mess luôn 

```
 <%= render partial: "field_type", collection: @field.field_types %>
 ```

**Comment by nguyentrieutien1** [comment]
ready

**Pull request merged**